nnef/src/framework.rs

```diff
@@ -80,11 +82,17 @@ impl Nnef {
         self.write_to_tar(model, w)?;
         Ok(())
     }
-
+    
```

Comment:
I think we should try to make it possible to give more control about the nesting of compression and networks: decompressing at loading time is not negligible, and compressing neural networks is not super effective in the first place. And here we are even inducing double compression...